lua/bookmarks/domain/bookmark_list.lua

```diff
@@ -1,11 +1,12 @@
-local bookmark_scope = require("bookmarks.domain.bookmark")
+local M = require("bookmarks.domain.bookmark")
```

Comment:
redefined local `M`.  line 1 and line 11

Reply:
oh, after merge, I replace all the bookmarks list name to M